{"name": "test_global_quote_without_symbol_parameter", "status": "broken", "statusDetails": {"message": "TypeError: assert_equal() got an unexpected keyword argument 'msg'", "trace": "self = <test_global_quote.Test_GlobalQuote testMethod=test_global_quote_without_symbol_parameter>\n\n    def test_global_quote_without_symbol_parameter(self):\n        self.logger.info(\"test_time_series_daily_invalid_symbol started\")\n        url = self.base_url+\"function=GLOBAL_QUOTE&symbol=&apikey=\"+self.apiKey\n        self.logger.info(\"url : \" + url)\n        response = requests.get(url)\n        response_json = response.json()\n        self.logger.info(\"response : \" + response.text)\n        # response assertion\n>       self.assertion.assert_equal(response_json['Error Message'],\n                         \"Invalid API call. Please retry or visit the documentation (https://www.alphavantage.co/documentation/) for GLOBAL_QUOTE.\",\n                         msg=\"response not meeted expected error\" + response_json['Error Message'])\nE       TypeError: assert_equal() got an unexpected keyword argument 'msg'\n\nTests/test_global_quote.py:81: TypeError"}, "attachments": [{"name": "log", "source": "37d10b04-cce4-4e41-9e94-89bf5b58bec4-attachment.txt", "type": "text/plain"}], "start": 1611645538956, "stop": 1611645539459, "uuid": "751b296f-71fa-4cb6-9b4e-d3139bc6ce77", "historyId": "b625d0959969e1fdba1e54a9f3077f5a", "testCaseId": "c17726e893171150a5be7d14da36291f", "fullName": "Tests.test_global_quote.Test_GlobalQuote#test_global_quote_without_symbol_parameter", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_global_quote"}, {"name": "subSuite", "value": "Test_GlobalQuote"}, {"name": "host", "value": "san-ThinkPad-T480"}, {"name": "thread", "value": "268115-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_global_quote"}]}